tests/auth.js

```diff
@@ -0,0 +1,143 @@
+var test = require('tape')
+var path = require('path')
+var spawn = require('./helpers/spawn')
+var help = require('./helpers')
+var authServer = require('./helpers/auth-server')
+
+var dat = path.resolve(path.join(__dirname, '..', 'bin', 'cli.js'))
+var baseTestDir = help.testFolder()
+
+var port = process.env.PORT || 3000
+var SERVER = 'http://localhost:' + port
+
+authServer(port, function (server) {
+  test('auth - whoami works when not logged in', function (t) {
+    // cmd: dat clone <link> (same thing again as above)
```

Comment:
We should probably just take all these comments out. I kept making the same mistake copy/pasting.